hip/static/styles/pages/data_report_detail_page.scss

```diff
@@ -1,6 +1,5 @@
 @import "../mixins";
 
-.reports-detail-page-mainarea-hip {
-  @include headers;
-  @include paragraphs;
+.reports-detail-page-hip {
```

Comment:
It looks like `reports-detail-page-hip` is not used in any HTML template, so these styles aren't applied on the report detail page

Reply:
Looks like I overwrote this class when I was copying and pasting. Great catch.

Reply:
DONE